drivers/eaton-pdu-marlin-mib.c

```diff
@@ -582,7 +582,7 @@ static snmp_info_t eaton_marlin_mib[] = {
 		NULL, SU_FLAG_STATIC | SU_FLAG_OK | SU_TYPE_DAISY_1, NULL, NULL },
 	/* inputPowerCapacity.0.1 = INTEGER: 2300 */
 	/* FIXME: RFC on key name is needed when backporting to NUT upstream */
-	{ "input.power.nominal", 0, 1.0, ".1.3.6.1.4.1.534.6.6.7.3.5.1.9.%i.1",
+	{ "input.realpower.nominal", 0, 1.0, ".1.3.6.1.4.1.534.6.6.7.3.5.1.9.%i.1",
```

Comment:
So, update the `scripts/DMF/dmfsnmp/eaton-pdu-marlin-mib.dmf` with `power => realpower` change and bump the MIB version as appropriate. Or regenerate and use the result.

Reply:
gone the manual way, I've some issues with the regen that need audit

Reply:
Seems the MIB version is not updated (see the `mib2nut` tag in the end of file).